install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -12,262 +12,728 @@ toc::[]
 
-To solve this issue, you can add the group *1234* to `*SupplementalGroups*` as
-in the following example:
+# id nobody
+uid=99(nobody) gid=99(nobody) groups=99(nobody)
+----
+====
 
+[NOTE]
 ====
+In the above, the owner is 99 (*nobody*), but the suggestions and examples in
+this topic apply to any non-root owner.
+====
+
+The *_/opt/nfs/_* export is accessible by UID *99* and the group *5555*. In
+general, containers should not run as root, so in this NFS example, containers
+which are not run as UID *99* or are not members the group *5555* will not be
+able to access the NFS export.
 
+Often, the SCC matching the pod does not allow an arbitrary user ID to be
+specified, thus using supplemental groups is a more flexible way to grant
+storage access to a pod. For example, to grant NFS access to the export above,
+the group *5555* can be defined in the pod definition:
+
+====
 [source,yaml]
 ----
 apiVersion: v1
 kind: Pod
-metadata:
-  name: nfs-web
+...
 spec:
   containers:
-    - name: web
-      image: nginx
-      ports:
-        - name: web
-          containerPort: 80
-      volumeMounts:
-          - name: nfs
-            mountPath: "/usr/share/nginx/html"
-  securityContext:
-    supplementalGroups: [1234]
+  - name: ...
+    volumeMounts:
+    - name: nfs <1>
+      mountPath: /usr/share/... <2>
+  securityContext: <3>
+    supplementalGroups: [5555] <4>
   volumes:
-    - name: nfs
-      nfs:
-        server: example.com
-        path: "/nfsexport"
+  - name: nfs <1>
+    nfs:
+      server: <nfs-server-ip-or-host>
+      path: /opt/nfs <5>
 ----
+<1> Name of the volume mount. Must match the name in the `*volumes*` section.
+<2> NFS export path as seen in the container.
+<3> Pod global security context. Applies to all containers in the pod. Each
+container can also define its `*securityContext*`, however group IDs are global
+to the pod and cannot be defined for individual containers.
+<4> Supplemental groups, which is an array of IDs, is set to 5555. This grants group
+access to the export.
+<5> Actual NFS export path on the NFS server.
 ====
 
-All containers in the above pod will be members of the group *1234* and will
-have access to the volume regardless of what user the container runs as. This is
-useful for volumes which do not allow you to change the export's ownership on
-the client, such as NFS and GlusterFS.
+All containers in the above pod (assuming the matching SCC or project allows the
+group *5555*) will be members of the group *5555* and have access to the volume,
+regardless of the container's user ID. However, the assumption above is
+critical. Sometimes, the SCC does not define a range of allowable group IDs but
+requires group ID validation (due to `*supplementalGroups*` set to *MustRunAs*;
+note this is not the case for the *restricted* SCC). The project will not likely
+allow a group ID of 5555, unless the project has been customized for access to
```

Comment:
s/5555/*5555*  (to be consistent with the rest of the instances in the paragraph)